docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -36,6 +36,53 @@ Additional information, including the specific error and a stack trace, is provi
 [role="screenshot"]
 image::images/synthetic-app-error.png[Synthetics app error]
 
+Select a step name to view the waterfall chart for the synthetic monitor.
+
+[role="screenshot"]
+image::images/synthetics-waterfall.png[Synthetics waterfall chart]
+
+The waterfall chart provides a visualization of every request the page made when
+a user executed it. Each line in the chart represents an HTTP network request and
+helps you quickly identify what resources are taking the longest to load and in what
+order they are loading.
+
+The colored bars within each line indicate the time spent per resource. Each color
```

Comment:
Typo? `indicates`

Reply:
My understanding is that `indicates` would be used for a singular subject. For this case, colored bars , `indicate` would be the correct usage.